schema-engine/sql-introspection-tests/tests/tables/mod.rs

```diff
@@ -461,7 +461,7 @@ async fn default_values(api: &mut TestApi) -> TestResult {
     Ok(())
 }
 
-#[test_connector(tags(Postgres), exclude(CockroachDb, Postgres14, Postgres15))]
+#[test_connector(tags(Postgres), exclude(CockroachDb, Postgres14, Postgres15, Postgres16))]
```

Comment:
Any insight why this should be excluded by chance that you can add as comment along the way?

Reply:
No, I don't remember why it's like this at the moment.